python_binding/lsm/_data.py

```diff
@@ -357,12 +357,26 @@ class System(IData):
     STATUS_PREDICTIVE_FAILURE = 1 << 4
     STATUS_OTHER = 1 << 5
 
-    def __init__(self, _id, _name, _status, _status_info, _plugin_data=None):
+    def __init__(self, _id, _name, _status, _status_info, _plugin_data=None,
+                 _fw_version=''):
```

Comment:
As I'm looking at doing something similar in the Disk class...do you need to change the @default_property elements somehow up above?